server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -278,4 +297,28 @@ private HaltException upsertUser(Request request, Response response) throws Json
     private Username extractUsername(Request request) {
         return Username.of(request.params(USER_NAME));
     }
+
+    private boolean isForced(String forceParam) {
+        return Optional.ofNullable(forceParam)
+                .map(String::trim)
+                .map(this::parseForceParam)
+                .orElse(false);
+    }
+
+    private boolean parseForceParam(String forceParam) {
+        if (forceParam.equalsIgnoreCase(Boolean.TRUE.toString())
+                || forceParam.equalsIgnoreCase(Boolean.FALSE.toString())) {
+            return Boolean.parseBoolean(forceParam);
+        }
+        throw throw400withInvalidArgument("Invalid activate query parameter");
```

Comment:
Invalid force query parameter.

You get a test about "force query parameter parsing"?

Reply:
I think I asked for `curl -XGET url/yyyy?force` not `curl -XGET url/yyyy?force=true`

Force query parameter is here => force
Force query parameter is not here => not force